modules/ztp-image-based-upgrade-prep-oadp.adoc

```diff
@@ -0,0 +1,136 @@
+// Module included in the following assemblies:
+// * edge_computing/image-based-upgrade/cnf-preparing-for-image-based-upgrade.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ztp-image-based-upgrade-creating-backup-resources-with-ztp_{context}"]
+= Creating OADP resources for the image-based upgrade with GitOps ZTP
+
+You need to prepare your OADP resources to restore your application after an upgrade.
+
+.Prerequisites
+
+* Provision one or more managed clusters with GitOps ZTP.
+* Log in as a user with `cluster-admin` privileges.
+* Generate a seed image from a compatible seed cluster.
+* Create a separate partition on the target cluster for the container images that is shared between stateroots. For more information about, see _Additional resources_.
+* Deploy a version of {lcao} that is compatible with the version used with the seed image.
+* Install the OADP Operator, the `DataProtectionApplication` CR and its secret on the target cluster.
+* Create an S3-compatible storage solution and a ready-to-use bucket with proper credentials configured. For more information, see Additional resources.
+
+.Procedure
+
+. Ensure that your Git repository that you use with the ArgoCD policies application contains the following directory structure:
++
+--
+[source,terminal]
+----
+├── source-crs/
+│   ├── ibu/
+│   │    ├── ImageBasedUpgrade.yaml
+│   │    ├── PlatformBackupRestore.yaml
+│   │    ├── PlatformBackupRestoreLvms.yaml
+├── ...
+├── ibu-upgrade-ranGen.yaml
+├── kustomization.yaml
+----
+
+[IMPORTANT]
+====
+The `kustomization.yaml` file must be located in the same directory structure as shown above to reference the `PlatformBackupRestore.yaml` manifest.
```

Comment:
Avoid directional language like "above" for accessible writing. Might say that it's "previously" shown, refer to the code block specifically, or just cut the note to "The `kustomization.yaml` file must be in the root of the `source-crs` directory."